sdk/typescript/src/types/transactions.ts

```diff
@@ -154,7 +154,12 @@ export type SuiExecuteTransactionResponse =
 
 export type GatewayTxSeqNumber = number;
 
-export type GetTxnDigestsResponse = [GatewayTxSeqNumber, TransactionDigest][];
+export type GetTxnDigestsResponse = TransactionDigest[];
+
+export type TransactionPage = {
```

Comment:
```suggestion
export type PaginatedTransactionDigests = {
```

nit: `PaginatedTransactionDigests` might be a better name